### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Zeph180**
- Set up a new project with webpack 
- create an index.html file 
- Create an index.js file and set an array of some simple to do tasks
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- Create a style.css and set rules for the To Do List.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
+const list = document.getElementById('list');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
- element.classList.add('hello');
+const toDos = [
+  {
+    description: 'Wash car',
+    completed: false,
+  },
```
Comment (algerina) [inline src/index.js]: - [ ] Kindly add an index to each todo.

**Review by algerina — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

### Hello @bhskr44   ,
 ✋🏼 Nice done I am impressed. Your project is great. There are just some small changes to add.

HIGHLIGHTS
- Your Project looks great :heavy_check_mark: 
- You have a descriptive PR :heavy_check_mark:   


## Required changes :


- [ ] Please, check the comments under the review.

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: src/index.js (hunk 1 above)
Comment (bhskr44) [reply src/index.js]: Hi @algerina , Thank you for the review It was great effort. I think you should have tagged @Zeph180.
@Zeph180 Please go through the changes requested.
Thanks.

Comment (Zeph180) [reply src/index.js]: Alright im on it @bhskr44  @algerina

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Zeph180, 👋 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

### Changes Required

- [x] _please check the inline review given if there is any_

### Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**